backend/src/main/java/com/huellapositiva/application/controller/ESALContactPersonApiController.java

```diff
@@ -83,4 +97,45 @@ public JwtResponseDto registerContactPerson(@Validated @RequestBody RegisterESAL
     public ProposalResponseDto getMember(@PathVariable Integer id) {
         throw new UnsupportedOperationException();
     }
+
+    @Operation(
+            summary = "Upload contact person Photo",
+            description = "Upload contact person Photo to profile",
+            tags = "user",
+            parameters = {
+                    @Parameter(name = "X-XSRF-TOKEN", in = ParameterIn.HEADER, required = true, example = "a6f5086d-af6b-464f-988b-7a604e46062b", description = "For take this value, open your inspector code on your browser, and take the value of the cookie with the name 'XSRF-TOKEN'. Example: a6f5086d-af6b-464f-988b-7a604e46062b"),
```

Comment:
The CSRF token has to be provided both as a cookie and a query parameter. Originally it was provided through a header but in was changed to satisfy frontend requirements

See Postman collection in the project for its usage
`
ParameterIn.QUERY
`

Reply:
You mean change name = "X-XSRF-TOKEN" -> name = "_csrf" and  ParameterIn.Header -> ParameterIn.Query?